content/en/docs/Responses/AlbumID3.md

```diff
@@ -127,6 +128,7 @@ description: >
 | `originalReleaseDate` | [`ItemDate`](../itemdate) | No |   **Yes**   | Date the album was originally released. |
 | `releaseDate` | [`ItemDate`](../itemdate)  | No |   **Yes**   | Date the specific edition of the album was released. *Note:* for files using ID3 tags, releaseDate should generally be read from the **TDRL** tag. Servers that use a different source for this field should document the behavior. |
 | `isCompilation` | `boolean` | No |  **Yes**    | True if the album is a compilation. |
+| `explicitStatus` | `string` | No |  **Yes**    | Returns "Explicit" if at least one song is Explicit, "Clean" if no song is Explicit and at least one is "Clean" else "". |
```

Comment:
Should we use lowercase for the option: `explicit` or `clean`? Or noting it is case insensitive? IMO TitleCase can lead to unnecessary confusion.

Reply:
I hesitated with just E and C or int values. I have no strong feelings, but yes if we keep the full values, lowercase is better.